driver-sync/src/test/functional/com/mongodb/client/ClientSideEncryption25LookupProseTests.java

```diff
@@ -72,6 +72,7 @@ protected ClientEncryption createClientEncryption(final ClientEncryptionSettings
 
     @BeforeEach
     public void setUp() {
+        assumeFalse(() -> true); // https://jira.mongodb.org/browse/JAVA-5837
```

Comment:
This is required as at the schema version 1.21 commit of the specifications repo we doesn't contain have the test files yet